core/quorum_recv.go

```diff
@@ -658,6 +658,123 @@ func (c *Core) quorumNFTConsensus(req *ensweb.Request, did string, qdc didcrypto
 	return c.l.RenderJSON(req, &consensusReply, http.StatusOK)
 }
 
+func (c *Core) quorumFTConsensus(req *ensweb.Request, did string, qdc didcrypto.DIDCrypto, cr *ConensusRequest) *ensweb.Result {
```

Comment:
What specific operations are you doing here when compared to `quorumRBTConsensus` method?

Reply:
Pinning service checks is not in FT consensus. No extra operation is doing in `quorumFTConsensus` other than from `quorumRBTConsensus`.